### ISSUE TITLE
Unable to publish on npmjs

### ISSUE THREAD

**Issue by hnagrath09** [dopplr-labs/tail-kit#31]
**Describe the bug**
`npm publish` command is not able to publish the latest changes on npmjs.com. This is happening after adding `Form` component. Error message is telling that Form component has circular dependency with it's other compound component elements.

**To Reproduce**
1. Pull the latest develop branch
2. Use `npm publish` command
3. Bump the version

**Comment by hnagrath09** [comment]
Circular dependency issue is resolved using following solution in index.ts

```
import { Form, LayoutOptions } from './form'
import { FormItem } from './form-item'

export default Object.assign(Form, { Item: FormItem, Layout: LayoutOptions })
```

But still not able to publish after exporting Form from `src/index.ts`.
Now facing problem due to uglifyJS
<img width="1792" alt="Screenshot 2021-05-07 at 9 52 43 PM" src="https://user-images.githubusercontent.com/33781906/117480024-ad1c8680-af7e-11eb-837b-60cccd74545b.png">

**Comment by hnagrath09** [comment]
This issue is fixed in v0.1.13 by replacing `rollup-plugin-uglify` with `rollup-plugin-terser`
